src/packages/admin-ui-components/src/utils/utils.ts

```diff
@@ -6,3 +8,18 @@ export const isNumeric = (item: unknown): boolean => {
 	}
 	return false;
 };
+
+export const getOrderByQuery = (sort?: SortField[]) => ({
+	...(sort
+		? sort.reduce((acc, { field, direction }) => ({ ...acc, [field]: direction }), {})
+		: { id: 'ASC' }),
+});
+
+export const getAndFiltersQuery = (filters: FieldFilter) => {
```

Comment:
I wonder if a better name for this is `wrapFilterWithAndOperator`

Reply:
The function has now been renamed to `wrapFilterWithAndOperator `.